posts/JHipster-Conf-2-Hottest-JHipster-Day-of-the-Year.md

```diff
@@ -0,0 +1,59 @@
+
+# JHipster updates 
+
+## JHipster 6
+
+JHipster 6 was released last month with all its major components upgraded to their latest release (Spring Boot, Angular, React, JUnit 5, etc).
+Other important changes that comes with version 6:
+- JDK 11 with JDK 8 compatibility
+- Default packaging changed to Jar (make Jar not War)
+- Bootswatch theme selection
+
+## Cloud support
+
+Overall, the cloud support is improved with the latest Spring Cloud release train: Greenwhich. Helm is now supported with its own sub-generator and the Kubernetes/Istio support was also improved.
+
+## Vue.js
+
+[Vue.js](https://vuejs.org/) is the new trending framework in the Javascript front-end world, and its simplicity lets you build web applications pretty fast. I really recommend trying the official [Vue.js blueprint](https://github.com/jhipster/jhipster-vuejs), here a [blog post](https://blog.ippon.tech/creating-a-modern-web-app-using-vuejs-and-spring-boot-with-jhipster/) that explains how Vue.js is different than Angular and React. The JHipster core team is also discussing on merging the blueprint in the main project which will simplify its usage.
+
+## Project metrics
+
+With 100,000 installations per month and more than 2 million in total, JHipster popularity is having a steady grow. The [main website](https://www.jhipster.tech/) gets 90,000 unique visitors per month and the project on [Github](https://github.com/jhipster/generator-jhipster) has more than 14,000 stargazers!
+
+## Miscellaneous
+
+If you're new to the JHipster world, you can follow a serie of [free tutorials here](https://github.com/jhipster/jhipster-guides). All you need is a Google account and you will be able to play with JHipster using Google Cloud Shell.
+
+In case you did not know, the whole JHipster's artwork was updated with a new logo and a whole JHipster family. More details can be found on the [official page](https://www.jhipster.tech/artwork/).
+
+# Roadmap
+
+## JDL centrict configuration
+
+![JDL](https://github.com/falydoor/blog-usa/blob/jhipster-conf-2/images/2019/07/jdl.png)
+
+## Prettier for Java
+
+[Prettier](https://prettier.io/) is an opinionated code formatter and it helps the team having an unified code formatter for the whole project. It is used by JHipster for the javascript/typescript files but Prettier doesn't support Java right now. So instead of waiting, the team decided to [create the plugin](https://github.com/jhipster/prettier-java)!
```

Comment:
```suggestion
[Prettier](https://prettier.io/) is an opinionated code formatter and it helps the team by having a unified formatter for the whole project. It is used by JHipster for the javascript/typescript files but Prettier doesn't support Java right now. So, instead of waiting, the team decided to [create the plugin themselves](https://github.com/jhipster/prettier-java)!
```